ui/etc/schema.yml

```diff
@@ -87,7 +87,7 @@ mapping:
 
       replication_group_defaults:
         type: map
-        required: True
+        allowempty: True
```

Comment:
According to the [pykwalify docs](http://pykwalify.readthedocs.io/en/unstable/validation-rules.html#allowempty):

```
Allowempty

Only applies to mapping.
If True, the map can have keys which are not present in the schema, and these can map to anything.
Any keys which are specified in the schema must have values which conform to their corresponding constraints, if they are present.

Default is False.
```

With `allowempty: True` we may validate typos in the keys. 

I think a better solution would be just setting `required: False` or just removing that line since False is the default value. I'd rather explicitly set it to False just in case the specs change in newer versions.

With `required: False`, if the key is omitted, the schema will validate it, but if the key is present, it will check the content strictly and not allow undocumented keys.